Dockerfile

```diff
@@ -253,12 +253,16 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 # 1. Install programs via pip
 
-# Install jaxlib on linux/arm64
+# Install jaxlib & jax on linux/arm64
 # jaxlib, an evofr dependency, does not have official pre-built binaries for
 # linux/arm64. A GitHub user has provided them in a fork repo.
 # https://github.com/google/jax/issues/7097#issuecomment-1110730040
+# Also hard-coding jax version here since it needs to match the jaxlib version
+# The minimum version requirement for jaxlib is checked at runtime rather than by pip
+# https://jax.readthedocs.io/en/latest/jep/9419-jax-versioning.html#how-are-jax-and-jaxlib-versioned
```

Comment:
It kinda sucks that we have to rely on an unofficial distribution of pre-built binaries plus manage the "peer dependencies" here, all for a dependency of evofr. Sadly this is the price we pay for supporting mutli-arch images.

I'm guessing the `linux/amd64` image variant works fine since the `jax`/`jaxlib` versions are resolved to when running `pip3 install evofr`. However, usage of that image might be noticeably slower on Apple silicon.